.circleci/config.yml

```diff
@@ -169,14 +175,14 @@ jobs:
       - run: npm rebuild node-sass
       - run: yarn build-for next --progress false --base-href /pr$CIRCLE_PULL_REQUEST_NUMBER-$SHORT_GIT_HASH/ --output-path dist/ngrx.io/pr$CIRCLE_PULL_REQUEST_NUMBER-$SHORT_GIT_HASH/ && yarn copy-404-page
       - run: cp -rf src/extra-files/next/. dist/ngrx.io/pr$CIRCLE_PULL_REQUEST_NUMBER-$SHORT_GIT_HASH/
-      - run: yarn --cwd ../../ install && yarn --cwd ../../ run deploy:preview
+      - run: yarn --cwd ../../ install --ignore-engines && yarn --cwd ../../ run deploy:preview
```

Comment:
Added `--ignore-engines` because we have a version mismatch between the docs and the rest of the packages, otherwise the install command would fail.